test/Microsoft.AspNet.Mvc.IntegrationTests/TryValidateModelIntegrationTest.cs

```diff
@@ -140,7 +140,7 @@ public void TryValidateModel_CollectionsModel_ReturnsErrorsForInvalidProperties(
         {
             var controller = new TestController();
             controller.ControllerContext = new ControllerContext(operationContext.ActionContext);
-            controller.ObjectValidator = ModelBindingTestHelper.GetObjectValidator();
+            controller.ObjectValidator = ModelBindingTestHelper.GetObjectValidator(operationContext.MetadataProvider);
```

Comment:
I see now: The comparison branch in this PR must not be up-to-date so this shows up as a change here.